spec/spec_helper.rb

```diff
@@ -69,6 +85,22 @@ def stub_github_repo(repo_name, feature_branches: [], contents: [])
         }.to_json,
       )
   end
+
+  pull_request_branches.each do |branch|
+    stub_request(:get, "https://api.github.com/repos/alphagov/#{repo_name}/pulls?head=alphagov:#{branch}&per_page=100")
+    .to_return(
+      status: 200,
+      headers: { "Content-Type": "application/json" },
+      body: [
+        {
+          "head": {
+            "label": "alphagov:#{branch}",
+            "ref": branch,
+          },
+        },
+      ].to_json,
+    )
```

Comment:
You can just reuse your method here:

```suggestion
    stub_github_get_pull_requests(repo_name, branch)
```